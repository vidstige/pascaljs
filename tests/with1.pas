program with1;
type
    TPerson = record
        name: String;        
        age: Integer;
    end;
var
    owner: TPerson;
begin
    with owner do
    begin
        name := 'Bilbo Baggins';
        age := 111;        
    end;
    WriteLn('name: ', owner.name, ', age: ', owner.age);
end.