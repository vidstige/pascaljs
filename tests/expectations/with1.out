name: Bilbo Baggins, age: 111
